packages/dx-grid-core/src/plugins/grid-core/computeds.js

```diff
@@ -0,0 +1,27 @@
+export const gridRows = data => data.map((row, index) => ({ row, defaultRowId: index }));
+
+export const gridRowIdGetter = getRowId => (gridRow) => {
+  if (getRowId) {
```

Comment:
It should be resolved outside the result function. We don't have to check this on per-row call.